src/stcal/jump/jump.py

```diff
@@ -610,10 +637,26 @@ def extend_saturation(
                 (0, 0, 22),
                 -1,
             )
+            #  Create another non-extended ellipse that is used to create the
+            #  persist_jumps for this integration. This will be used to mask groups
+            #  in subsequent integrations.
             sat_ellipse = image[:, :, 2]
             saty, satx = np.where(sat_ellipse == 22)
             outcube[grp:, saty, satx] = sat_flag
-    return outcube
+            persist_image = cv.ellipse(
+                persist_image,
+                (round(ceny), round(cenx)),
+                (round(ellipse[1][0] / 2), round(ellipse[1][1] / 2)),
+                alpha,
+                0,
+                360,
+                (0, 0, 22),
+                -1,
+            )
+            persist_ellipse = persist_image[:, :, 2]
+            persist_saty, persist_satx = np.where(persist_ellipse == 22)
```

Comment:
What is special about 22 here?  A comment or declaring a descriptive variable with the value 22 would be good here.

Reply:
I updated the comments in this section. Let me know, if you think it needs more details.

Reply:
No real reason that I can remember. There needs to be a number that I can use np.where to find the pixels that are affected. I'm sure there's a more elegant way to do it.

Reply:
I think it would be fine to make a comment saying this number could change and that this number was determined though trial and error.  But more importantly what this number is supposed to represent.  Numbers with no reference or description leaves future programmers to try to figure out what that number represents and why that number was chosen.

Reply:
The value of 22 is arbitrary. It just needs to be something that can be tested for to create the mask of pixels within the ellipse.